src/features/element-hiding.js

```diff
@@ -318,11 +370,12 @@ export default class ElementHiding extends ContentFeature {
      */
     applyRules (rules) {
         const timeoutRules = extractTimeoutRules(rules)
+        // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
+        const clearCacheTimer = unhideTimeouts.at(-1) + 100
```

Comment:
In case we change the ordering or which has the larger number we should consider joining all timers and finding the max. Something like:

```suggestion
        const clearCacheTimer = unhideTimeouts.concat(hideTimeouts).reduce((a,b) => Math.max(a,b), 0) + 100
```